includes/core/server/request/class-get-account.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\Get_Account.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WCPay\Core\Server\Request;
+use WC_Payments_API_Client;
+
+/**
+ * Request class for getting intents.
+ */
```

Comment:
Please update this class description :)

Reply:
😬 - good point. Addressed 4c3cb3d834ea0db76bf3c676ba9a15d36ed7a74d